### PR TITLE
iOS Support Foreground for Input.Toggle & Input.ChoiceSet

### LINKED ISSUE #3542: [iOS][Dark theme: Text Color] [Text color for toggle switch in dark mode is not being set properly]
# Platform

* iOS

# Author or host

Are you an author (like sending something to Outlook), or a host that is rendering your own cards? If host, please specify the corresponding host (app) name.

Host: Teams App

# Version of SDK

1.2

# Details
The text color in dark theme for toggle switch is not being set correctly.

### PR DESCRIPTION
**Pull request by nesalang**
## Related Issue
Fixed #3542 

## Description
Input.Toggle and Input.ChoiceSet's text foreground didn't adapt parent container style as it should, so when the parent's background changes to very dark according to container style, the foreground will stay black.
This PR alows Input.ChoiceSet and Input.Toggle's text to adjust to parent container style. ForeGround Color will always choose Default since the inputs do not offer color selection options.

## How Verified
1. tested json payloads with dark theme.

Simulator Screen Shot - iPhone 8 - 2019-10-30 at 16 49 28

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/AdaptiveCards/pull/3552)

### CODE REVIEW

**Comment by shalinijoshi19** [comment]
@almedina-ms is the AdaptiveCards-Xamarin pipeline supposed to be triggered for PRs/CI at this point? it's causing build failures looks like here

**Comment by almedina-ms** [comment]
> 
> 
> @almedina-ms is the AdaptiveCards-Xamarin pipeline supposed to be triggered for PRs/CI at this point? it's causing build failures looks like here

@shalinijoshi19 , the pipeline is disabled, I think when the PR was first uploaded the pipeline was still enabled so re-running the build definitions should remove the failure

image

**Comment by shalinijoshi19** [comment]
Ok I just indirectly queued a rerun by triggering a code update with master on the PR and now its not picking up the Xamarin build definition anymore.. 

---
In reply to: [549779140](https://github.com/microsoft/AdaptiveCards/pull/3552#issuecomment-549779140) [](ancestors = 549779140)

File: source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRInputChoiceSetRenderer.mm [hunk 1]
```diff
@@ -54,7 +54,7 @@ - (UIView *)render:(UIView<ACRIContentHoldingView> *)viewGroup
         dataSource = [[ACRChoiceSetViewDataSourceCompactStyle alloc] initWithInputChoiceSet:choiceSet rootView:rootView];
         [choiceSetView setSeparatorStyle:UITableViewCellSeparatorStyleNone];
     } else {
-        dataSource = [[ACRChoiceSetViewDataSource alloc] initWithInputChoiceSet:choiceSet];
+        dataSource = [[ACRChoiceSetViewDataSource alloc] initWithInputChoiceSet:choiceSet WithHostConfig:config];
```
Comment (shalinijoshi19) [inline source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRInputChoiceSetRenderer.mm]: >ceSet:choiceSet WithHostConfig:co [](start = 74, length = 33)

Can we add unit tests for this API/call at all ? or is that not written to support unit tests at this point? If not can we add the sample card + host config you were working with to the samples folder if it doesnt already exist? thanks! Basically whatever it is we need to do to ensure this is part of (eventual automated) test suites/plan that we have.

**Comment by shalinijoshi19** [comment]
@nesalang, this needs to go into release/1.2 as well. Please ensure either via a git cherry-pick or a manual port that we follow up wtih a commit to release/1.2 as well. Thanks

**Pull request merged**